cfgov/preprocessed/js/modules/util/breakpoint-state.js

```diff
@@ -17,7 +17,7 @@ function _inBreakpointRange( breakpointRange, width ) {
 
 /**
  * @returns {object} An object literal with Boolean
- * isMobile, isTablet, isDesktop, isWall, isSmall properties.
+ * isBpXs, isBpSm, isBpMed, isBpLg, isBpXl properties.
```

Comment:
This is where camelCase really falls down. But I still would capitalize the `XS` and `XL`, since nothing follows them.